scripts/packetReader.lua

```diff
@@ -416,6 +416,9 @@ packetReader.GetObjectPacketTables = function(packetType)
 
                 if isObjectPlayer then
                     player.hit = hit
+					player.hittingPid = nil			
+					player.hittingRefId = nil
+					player.hittingUniqueIndex = nil	
                 else
```

Comment:
Again, you use tabs for indentation, while the file uses 4 spaces.

Reply:
Thanks I updated it correctly